test/private/initializeGlobalState.m

```diff
@@ -0,0 +1,76 @@
+function [orig,cwd] = initializeGlobalState()
+% Initialize global state. Set working directory and various properties of
+% the graphical root to ensure reliable output of the ACID testsuite.
+% See #542 and #552
+%
+% 1. Working directory
+% 2. Bring get(0,'Default') in line with get(0,'Factory')
+% 3. Set specific properties, required by matlab2tikz
+    fprintf('Initialize global state...\n');
+    orig = struct();
+
+    %--- Initialize working directory
+    cwd = initializeWorkingDirectory();
+
+    %--- Extract user defined default properties and set factory state
+    default = get(0,'Default');
+    factory = get(0,'Factory');
+
+    f = fieldnames(default);    % fields of user's default state
+    for i = 1:length(f)
+        factory_property_name = strrep(f{i},'default','factory');
+        factory_property_value = factory.(factory_property_name);
+        orig.(f{i}).val = ...
+            swap_property_state(0, f{i}, factory_property_value);
+    end
+    clear default factory_property_name factory_property_value
```

Comment:
Why the `clear`? We should either document that OR remove it (I prefer the latter, if possible).